pkg/cloud/azure/actuators/cluster/reconciler.go

```diff
@@ -17,49 +17,259 @@ limitations under the License.
 package cluster
 
 import (
+	"fmt"
+	"hash/fnv"
+	"strings"
+
 	"github.com/pkg/errors"
 	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/actuators"
 	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/services"
 	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/services/certificates"
 	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/services/groups"
+	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/services/internalloadbalancers"
+	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/services/publicips"
+	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/services/publicloadbalancers"
+	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/services/routetables"
+	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/services/securitygroups"
+	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/services/subnets"
+	"sigs.k8s.io/cluster-api-provider-azure/pkg/cloud/azure/services/virtualnetworks"
 )
 
 // Reconciler are list of services required by cluster actuator, easy to create a fake
 type Reconciler struct {
-	scope           *actuators.Scope
-	groupsSvc       services.Service
-	certificatesSvc services.Service
+	scope            *actuators.Scope
+	certificatesSvc  services.Service
```

Comment:
can we omit the Svc suffix on these? certificates, groups, etc.

Reply:
I would like to keep it as Svc suffix to show the intent, else s.certificates could mean anything.

Reply:
maybe add a struct called services and make it a member of this struct.